api/bin/chainflip-ingress-egress-tracker/src/witnessing/state_chain.rs

```diff
@@ -14,6 +16,14 @@ use pallet_cf_ingress_egress::DepositWitness;
 use serde::Serialize;
 use utilities::rpc::NumberOrHex;
 
+#[derive(Serialize)]
+#[serde(untagged)]
+enum TransactionRef {
+	Bitcoin { hash: String },
```

Comment:
@marcellorigotti 
Explicit conversion to String is a bit of an anti-pattern. I think a better solution here is to use `BitcoinTransactionHash` and define `Serialize` on that struct. The *serialization* should reverse the byte order. Then we can use the same struct for the tx_ref and tx_out_id. 

IMO this is less confusing than storing two different tx references.